src/scr.c

```diff
@@ -1122,6 +1286,19 @@ static int scr_get_params()
       );
     }
   }
+  if (scr_my_rank_world == 0) {
+     scr_dbg(1, "SCR_CHECKPOINT_OVERHEAD=%f", scr_checkpoint_overhead);
+  }
+
+  if (scr_my_rank_world == 0) {
+     kvtree_print_mode(scr_groupdesc_hash, 4, KVTREE_PRINT_KEYVAL);
+  }
+  if (scr_my_rank_world == 0) {
+     kvtree_print_mode(scr_storedesc_hash, 4, KVTREE_PRINT_KEYVAL);
+  }
+  if (scr_my_rank_world == 0) {
+     kvtree_print_mode(scr_reddesc_hash, 4, KVTREE_PRINT_KEYVAL);
+  }
```

Comment:
Before each of the kvtree_print lines, let's add a note about what we're printing like so:
```suggestion
  if (scr_my_rank_world == 0) {
     scr_dbg(1, "Group descriptors:");
     kvtree_print_mode(scr_groupdesc_hash, 4, KVTREE_PRINT_KEYVAL);
  }
  if (scr_my_rank_world == 0) {
     scr_dbg(1, "Store descriptors:");
     kvtree_print_mode(scr_storedesc_hash, 4, KVTREE_PRINT_KEYVAL);
  }
  if (scr_my_rank_world == 0) {
     scr_dbg(1, "Redundancy descriptors:");
     kvtree_print_mode(scr_reddesc_hash, 4, KVTREE_PRINT_KEYVAL);
  }
```